scala-package/init/src/main/scala/org/apache/mxnet/init/Base.scala

```diff
@@ -37,7 +37,11 @@ object Base {
 
   @throws(classOf[UnsatisfiedLinkError])
   private def tryLoadInitLibrary(): Unit = {
-    val baseDir = System.getProperty("user.dir") + "/init-native"
+    // val baseDir = System.getProperty("user.dir") + "/init-native"
+    var baseDir = System.getProperty("user.dir") + "/init-native"
+    if (System.getenv().containsKey("BASEDIR")) {
+      baseDir = sys.env("BASEDIR")
+    }
```

Comment:
what are you expecting BASEDIR variable to be?

I think you should have a else
`else {
baseDir System.getProperty("user.dir")
}
baseDir = baseDir + "/init-native"
`

Reply:
BASEDIR locate in the pom file (set as an environment variable) which determine the current base directory.

Reply:
Done